package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "zonos-typescript-sdk",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "graphql": "^16.6.0",
+    "graphql-request": "^4.3.0",
+    "graphql-tag": "^2.12.6"
+  },
+  "engines": {
+    "node": "18"
+  },
```

Comment:
Considering our plugins user various lower versions of node (I believe), I wonder if maybe we should expand this range?

Reply:
Yah, that's why I remove the engine when preparing for publishing to npm

Reply:
The reason for this node version on package.json is just for unit testing to use built-in `fetch` of node 18